data-sources/amazon-chatbot/config.yml

```diff
@@ -0,0 +1,8 @@
+id: amazon-chatbot
+displayName: Amazon Chatbot
+description: Monitor Amazon Chatbot Access by sending Access logs to New Relic
+install:
+  primary:
+    link:
+      url: https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-chatbot-monitoring-integration/
```

Comment:
Looks like this doc page doesn't exist, are we waiting for a release on this one?